proto/astarteplatform/msghub/property.proto

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of Astarte.
+ *
+ * Copyright 2024 SECO Mind Srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+syntax = "proto3";
+
+package astarteplatform.msghub;
+
+import "astarteplatform/msghub/astarte_type.proto";
+
+/*  */
+enum Ownership {
+  DEVICE = 0;
+  SERVER = 1;
+}
+
```

Comment:
Since these are something we can share with a general interface not only to property, What do you think to move these into `interface.proto` file?

Reply:
I would move the Ownership for sure, not so sure about the other messages.